bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/flamingos2/Flamingos2.java

```diff
@@ -889,11 +911,27 @@ public double getSetupTime(ISPObservation obs) {
         return getSpectroscopySetupSec();
     }
 
+    // for ITC overheads
+    public double getSetupTime(Config conf) {
+        if (isImagingConfig(conf)) return getImagingSetupSec(conf);
+        return getSpectroscopySetupSec();
+    }
+
     /**
      * Is the instrument in imaging mode.
      */
+    private static boolean isImagingConfig(FPUnit fpu, Disperser disperser) {
+        return (fpu == FPUnit.FPU_NONE) && (disperser == Disperser.NONE);
+    }
+
+    private static boolean isImagingConfig(Config conf) {
+        return isImagingConfig(
+                (FPUnit) conf.getItemValue(FPUnit.KEY),
+                (Disperser) conf.getItemValue(Disperser.KEY));
+    }
+
     private boolean isImaging() {
-        return (_fpu == FPUnit.FPU_NONE) && (_disperser == Disperser.NONE);
+        return isImagingConfig(_fpu, _disperser);
```

Comment:
I really don't see an easy way to extract these changes from the model, since the original model methods have been changed to use them as well, so I'll leave them for now.